.changeset/errors-now-equal.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-6/core': major
+---
+
+Errors are now mostly context free - only list and field access-control is differentiable by the error message now
```

Comment:
This description doesn't explain the change well imo. Like what errors? What do you mean by context?